cstore_fdw.c

```diff
@@ -299,8 +307,9 @@ CStoreProcessUtility(Node * parseTree, const char *queryString,
 
 		if (dropStmt->removeType == OBJECT_EXTENSION)
```

Comment:
it does not. As soon as drop extension is called upon cstore_fdw, all triggers and functions are gone. Even though drop event might be fired, our trigger is not called.